### ISSUE TITLE
Wheeler - Quick Action Wheel Of Skyrim

### ISSUE THREAD

**Issue by Humankiin** [tiltedphoques/Mod-Compatibility#186]
# Mod information

Embeds a quick action wheel seamlessly into Skyrim's gameplay, providing a convenient interface for accurate and customized access and usage of spells, weapons, powers, shouts, potions, and poisons in the inventory.

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/97345

#### Dependencies

- Address Library for SKSE Plugins
  -  https://www.nexusmods.com/skyrimspecialedition/mods/32444

# Compatibility

Mod is not compatible. It hangs and then crashes on attempt to load or start a save.

# Additional Information

Tested with only Wheeler, SKSE, Address Library, and Skyrim Together.

**Comment by SkylerHaataja** [comment]
yeah just found this one out today. For me it worked at first but found out quickly that it causes CTDs sometimes and everytime when trying to start a new save. such a shame cuz it's a great mod, I couldn't get the mod wheeler ctd fix working either I assume because of the sse engine fixes requirement.

**Comment by miredirex** [comment]
Tested wheeler yesterday. This mod is a crashfest without Engine Fixes and its memory manager patch (`bOverrideMemoryManager = true` in the config). Frequent crashes even without Skyrim Together if EF isn’t installed or if you're using a non-vanilla .toml. If by the time you read this STR supports EF _with the memory patch_, running Wheeler should introduce no issues

**Comment by JJuicerr** [comment]
Now it is compatible as in the **Engine fixes v7** integrated with STR; Wheeler run well , but rarely Stuck/Freezes while bombarded with commands actions like opening character menu while using opening Wheeler then opening connection menu something like that other than that it worked fine on me, i let u know if problem occurs until now it running smooth. HAPPY MODDING AND GAMING!!!

**EDIT:** fixed rarely Stuck/Freezes use this steps to work as like mine for 1.6.1170 (steam) , 1.6.1179 (GOG) **[TESTED ON]**
### Mod Manager (recommended) [due to best for managing conflicts easily and have previews on conflicts]

> Vortex

### Mods (Major)

> dMenu by dTry - https://www.nexusmods.com/skyrimspecialedition/mods/97221

> dMenu NG by c0kAdam - https://www.nexusmods.com/skyrimspecialedition/mods/166751

> Wheeler by dTry - https://www.nexusmods.com/skyrimspecialedition/mods/97345

> Wheeler CTD-Fix by zombiesyard - https://www.nexusmods.com/skyrimspecialedition/mods/132074

> SSE Engine Fixes - https://www.nexusmods.com/skyrimspecialedition/mods/17230

**Comment by Thom-Ernst** [comment]
I would like to add to @JJuicerr there is now a new updated version to Wheeler and dMenu which have fixed many of the stability issues.
WHEELER - Refined https://www.nexusmods.com/skyrimspecialedition/mods/167380
dMenu NG https://www.nexusmods.com/skyrimspecialedition/mods/166751
This way the CTD fix is also no longer needed and it also contains additional features.
